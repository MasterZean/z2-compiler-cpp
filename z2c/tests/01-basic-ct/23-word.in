0u16
256u16
65534u16
65535u16