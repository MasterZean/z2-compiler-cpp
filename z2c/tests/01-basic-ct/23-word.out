0u;
256u;
65534u;
65535u;
